app/components/dex-create.jsx

```diff
@@ -7,14 +7,29 @@ import slug          from 'slug';
 import { AlertComponent } from './alert';
 import { ReactGA }        from '../utils/analytics';
 import { createDex }      from '../actions/dex';
+import { listGames }      from '../actions/games';
 
 const NATIONAL_ONLY_GAMES = ['x', 'y', 'omega_ruby', 'alpha_sapphire'];
 
 export class DexCreate extends Component {
 
   constructor (props) {
     super(props);
-    this.state = { error: null, game: 'sun', regional: false };
+    this.state = { error: null, game: 'ultra_moon', games: [], regional: false };
+  }
+
+  componentWillMount () {
+    this.reset();
+  }
+
+  reset (props) {
+    const { listGames } = props || this.props;
+
+    listGames()
```

Comment:
instead of calling this here, lets call it in the `profile` component. and since its setting `games` on the global state and wont be returning it, we can just call it and forget about it e.g.

```js
listGames();
```